pkg/kubelet/server/server.go

```diff
@@ -417,6 +419,20 @@ func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
 	}
 }
 
+// InstallDebuggingDisabledHandlers registers the HTTP request patterns that provide better error message
+func (s *Server) InstallDebuggingDisabledHandlers() {
+	glog.Infof("Adding debug disabled handlers to kubelet server.")
```

Comment:
Nit: drop this log message, it's sort of confusing

Reply:
Done!